awscli/handlers.py

```diff
@@ -142,3 +143,4 @@ def awscli_initialize(event_handlers):
         register_create_keys_from_csr_arguments)
     register_cloudfront(event_handlers)
     register_gamelift_commands(event_handlers)
+    EC2PageSizeInjector().register(event_handlers)
```

Comment:
I'd prefer we stick to the more common pattern of having a single module level function, `register_*`, that handles the registration.  It's by far the common pattern used here, and it will also make it easier to refactor in the future if we start to persue deferred event registration.